ui/app/AppLayouts/Wallet/popups/AddEditSavedAddressPopup.qml

```diff
@@ -180,7 +182,7 @@ StatusModal {
             mainModule.resolveENS(name, d.uuid)
         });
 
-        property var contactsModuleInst: root.store.profileSectionModuleInst.contactsModule
```

Comment:
Here the `root.store` is not `WalletStores.RootStore` but a `SharedStores.RootStore` in fact